karma.conf.maker.js

```diff
@@ -34,7 +34,6 @@ function newPluginsArray(browserstack) {
     'karma-es5-shim',
     'karma-mocha',
     'karma-chai',
-    'karma-requirejs',
```

Comment:
If you're removing `karma-requirejs` plug-in from this file, can it also be removed from the `package.json`?  I don't see it referenced anywhere else in the project.

Reply:
yes, removed